service/internal/auth/config.go

```diff
@@ -4,12 +4,12 @@ import "fmt"
 
 // AuthConfig pulls AuthN and AuthZ together
 type Config struct {
-	Enabled     bool `yaml:"enabled" default:"true" `
 	AuthNConfig `mapstructure:",squash"`
 }
 
 // AuthNConfig is the configuration need for the platform to validate tokens
 type AuthNConfig struct {
+	AllowNoDPoP       bool   `yaml:"allowNoDPoP" json:"allowNoDPoP" default:"true"`
```

Comment:
Does this change the behavior?

```suggestion
	AllowNoDPoP       bool   `yaml:"allowNoDPoP" json:"allowNoDPoP"`
```